.github/workflows/ci.yml

```diff
@@ -582,3 +582,33 @@ jobs:
       - run: sudo apt update && sudo apt install -y libhdf5-dev
       - shell: bash -l {0}
         run: python -m pytest modin/pandas/test/test_io.py::TestCsv
+
+  test-spreadsheet:
+    needs: [ lint-commit, lint-flake8, lint-black, test-api, test-headers ]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.6", "3.7", "3.8" ]
+        engine: ["ray", "dask"]
+    env:
+      MODIN_BACKEND: ${{matrix.engine}}
+      MODIN_EXPERIMENTAL: "True"
```

Comment:
```suggestion
```

Reply:
Should I also reduce the matrix? Adds 6 tests to the CI